README.md

```diff
@@ -581,6 +578,106 @@ This document is written to capture the conventions of writing styles for our pr
 ### Disclaimer
 It's important to note that client and project requirements always undo choices made in these guidelines
 
+### Formatting and Naming
+
+Names should be meaningful and not presentational:
+```css
+/* Bad examples */
+
+/* presentational with color and position reference */
+.button-orange {}
+.left-block {}
```

Comment:
To me it seems like were doing to different things here.
first example you first tell what the item is; a button, second example you first note a specification of the the item.

Reply:
Okay yeah then I agree. In case of buttons I think it's normal to make primary-button and secondary-button. Those could have variations. As far as I know we never standardize the way you can add variations by class. In React it's usually done by props.